app/controllers/frontend_log_controller.rb

```diff
@@ -33,6 +33,8 @@ class FrontendLogController < ApplicationController
     'IdV: personal key acknowledgment toggled' => :idv_personal_key_acknowledgment_toggled,
     'IdV: prepare submitted' => :idv_in_person_prepare_submitted,
     'IdV: prepare visited' => :idv_in_person_prepare_visited,
+    'IdV: selfie image added' => :idv_selfie_image_added,
```

Comment:
This one is added below on [this line](https://github.com/18F/identity-idp/pull/10244/files#diff-15d821e4605e7b3b601472200fd81280ef62b62514314079b8d44ac6561716edR53).

Were you seeing it show with spaces? I wonder if I missed something in https://github.com/18F/identity-idp/pull/10208.

Reply:
I tested locally with both SDK and upload on `main` and didn't see any warnings about missing logs. Maybe that's not the error you're running into though.

Reply:
Oh, interesting. I think I misunderstood the role of those two hashes. You're right that I don't actually care about `idv_selfie_image_added` for this PR. I just thought it was missing and added it.

I'm going to rework this PR shortly.

Reply:
The error that I'm trying to fix is because we're missing `idv_selfie_image_clicked`